src/routes/blogs/accelerating-phi-3/+page.svx

```diff
@@ -0,0 +1,131 @@
+
+Please watch this space for more updates on AMD, and additional optimization with ORT 1.18. Also, Check out our [Build Talk](https://build.microsoft.com/en-US/sessions/e6d21a49-2efb-4a39-8c26-f6eef1410c7a?source=sessions) in late May to learn more about this API!
+
+## Performance Metrics
+
+### DirectML:
+<div class="grid lg:grid-cols-2 gap-4 grid-cols-1">
+    <div class="col-span-1">
+We measured ONNX Runtime + DirectML performance of Phi-3 Mini (4k sequence length) quantized with AWQ and with a block size of 128 on Windows. Our test machine had an NVIDIA GeForce RTX 4090 GPU and an Intel Core i9-13900K CPU. As you can see in the table, DirectML offers high token throughput even at longer prompts and generation lengths.
+<br/><br/>
+DirectML lets developers not only achieve great performance but also lets developers deploy models across the entire Windows ecosystem with support from AMD, Intel and NVIDIA. Best of all, AWQ means that developers get this scale while also maintaining high model accuracy.
+<br/><br/>
+Stay tuned for additional performance improvements in the coming weeks thanks to optimized drivers from our hardware partners, along with additional updates to the ONNX Generate() API.
+
+    </div>
+    <div class="col-span-1">
+                
+<table>
+<tr><th>Prompt Length</th><th>Generation Length</th><th>Wall Clock tokens/s</th></tr>
+<tr><td>16</td><td>256</td><td>266.65</td></tr>
+<tr><td>16</td><td>512</td><td>251.63</td></tr>
+<tr><td>16</td><td>1024</td><td>238.87</td></tr>
+<tr><td>16</td><td>2048</td><td>217.5</td></tr>
+<tr><td>32</td><td>256</td><td>278.53</td></tr>
+<tr><td>32</td><td>512</td><td>259.73</td></tr>
+<tr><td>32</td><td>1024</td><td>241.72</td></tr>
+<tr><td>32</td><td>2048</td><td>219.3</td></tr>
+<tr><td>64</td><td>256</td><td>308.26</td></tr>
+<tr><td>64</td><td>512</td><td>272.47</td></tr>
+<tr><td>64</td><td>1024</td><td>245.67</td></tr>
+<tr><td>64</td><td>2048</td><td>220.55</td></tr>
+</table>
+
+    </div>
+</div>
+
+### CUDA:
+
+The table below shows improvement on the average throughput of the first 256 tokens generated (tps) for the Phi-3 Mini 128K Instruct ONNX model. The comparisons are for FP16 and INT4 precisions on CUDA, as measured on 1 A100 80GB GPU (SKU: [Standard_ND96amsr_A100_v4](https://learn.microsoft.com/en-us/azure/virtual-machines/ndm-a100-v4-series)).
+
+<div class="m-auto w50">
+<img src="./Phi3-Mini-128K-FP16CUDA.png" alt="Average throughput of Phi-3 Mini 128K Instruct ONNX model.">
+
+<i>Note: PyTorch Compile and Llama.cpp do not currently support the Phi-3 Mini 128K instruct model.</i>
+</div>
+<br/>
+
+
+The table below shows improvement on the average throughput of the first 256 tokens generated (tps) for Phi-3 Mini 4K Instruct ONNX model. The comparisons are for FP16 and INT4 precisions on CUDA, as measured on 1 A100 80GB GPU (SKU: Standard_ND96amsr_A100_v4).
```

Comment:
Do we have a Microsoft Learn link to this SKU?

Reply:
It's the [same SKU link](https://learn.microsoft.com/en-us/azure/virtual-machines/ndm-a100-v4-series) as above.